.github/workflows/node.js.yml

```diff
@@ -36,14 +36,20 @@ jobs:
       run: sudo chown -R $(whoami) /usr/share/nsis/Plugins/
       if: ${{ matrix.os == 'ubuntu-20.04' }}
 
+    - name: 'Install npm dependencies'
+      run: npm ci
+
     - name: 'Test'
-      run: npm test -- --reporter json --reporter-options output=./report-${{ matrix.os }}.json
+      id: npm-run-test
+      run: |
+        npm test -- --reporter json --reporter-options output=${{ runner.temp }}/report-${{ matrix.os }}.json
+        echo "::set-output name=path::${{ runner.temp }}/report-${{ matrix.os }}.json"
 
     - name: 'Publish Tests'
       uses: 'dorny/test-reporter@v1'
       # run this step even if previous step failed
       if: ${{ !env.ACT }} && (success() || failure())
       with:
         name: 'Tests ${{ matrix.os }}'
-        path: 'report-${{ matrix.os }}.json'
+        path: ${{ steps.npm-run-test.outputs.path }}
```

Comment:
It looks like the workflow is failing on this step. Can you double check the output path?

Reply:
This one is a bit trial & error. I'm using `act` to run actions locally, but I'm getting a different error on my machine.

Reply:
I made an [update](https://github.com/joncloud/makensis-action/commit/fa476b0e4ddb0700b695bbafdaf193c726c86529) to `.github/workflows/node.js.yml`, which I believe will correct the error we're getting.